src/fuzzer.ts

```diff
@@ -0,0 +1,111 @@
+import random from "crypto-random-bigint";
+import { InferType, IntType, SourceUnit, TypeNode, FunctionDefinition } from "solc-typed-ast";
+import { encodeFunctionCall } from "web3-eth-abi";
+import { runEVM, RunEVMResult } from "./evmc";
+
+export type FuzzingResult = {
+    runResult: RunEVMResult;
+    version: string;
+    encodedCall: string;
+    callParameters: number[];
+    functionName: string;
+};
+
+export class Fuzzer {
+    private functionCall: string;
+    private testFunction: FunctionDefinition;
+    private callParameters: number[];
+
+    constructor(
+        private unit: SourceUnit,
+        private testCallFunction: string,
+        private version: string
+    ) {
+        const testFunction = this.unit.vContracts[0].vFunctions.find(
+            (f) => f.name === this.testCallFunction
+        );
+        if (testFunction === undefined) {
```

Comment:
`testFunction` is actually always required, not just for eof. So make it always required in config.ts